docs/tutorials/vae.rst

```diff
@@ -371,25 +350,30 @@ Generate Images
 What we've done above is to define and learn the model. To see how it
 performs, we would like to let it generate some images in the learning process.
 For the generating process, we remove the observation noise, i.e.,
-the ``Bernoulli`` StochasticTensor. We do this by first change the model
-function a little to return one more instance,
-the direct output of the neural network (``x_logits``)::
+the ``Bernoulli`` StochasticTensor. 
+We do this by using the direct output of the nueral network (``x_logits``)::
 
-    @zs.reuse('model')
-    def vae(observed, n, x_dim, z_dim):
-        with zs.BayesianNet(observed=observed) as model:
+    @zs.meta_bayesian_net(scope="gen", reuse_variables=True)
+    def build_gen(x_dim, z_dim, n, n_particles=1):
+        bn = zs.BayesianNet()
+            ...
+        x_logits = tf.layers.dense(h, x_dim)
             ...
-            x_logits = layers.fully_connected(lx_z, x_dim, activation_fn=None)
-            x = zs.Bernoulli('x', x_logits, group_ndims=1)
-        # before change: return model
-        return model, x_logits
 
-Then we add a sigmoid function to it to get a "mean" image.
+Then we add a sigmoid function to it to get a "mean" image. 
+After that, we use the function: ``output`` to provide the value ``x_mean``,
+so that we can easily access it by using the function: ``observe``.
```

Comment:
we don't access it by observe, we access it by bn["x_mean"], but to get bn, we need to first cal meta_bn.observe.